apps/mark-scan/accessible-controller/src/controllerd.rs

```diff
@@ -476,6 +487,26 @@ mod tests {
             0xc8,
             0x37,
         ];
-        assert_eq!(handle_command(&data).unwrap().unwrap(), keyboard::Key::R);
+        assert_eq!(
+            handle_command(&data).unwrap().unwrap(),
+            (keyboard::Key::R, true)
+        );
+    }
+
+    #[test]
+    fn test_handle_command_success_no_shift() {
+        let data = [
+            0x30,
+            0x00,
+            0x02,
+            Button::Left as u8,
+            Action::Pressed as u8,
+            0xd8,
+            0x09,
```

Comment:
The last 2 values are the [CRC checksum](https://en.wikipedia.org/wiki/Cyclic_redundancy_check); the important difference in this test is that we are sending `Button::Left`, which as an arrow key requires no `Shift` key.